Formula/echidna.rb

```diff
@@ -53,6 +53,11 @@ def install
   test do
     system "truffle", "init"
 
+    # echidna does not appear to work with 'shanghai' EVM targets yet, which became the
+    # default in solc 0.8.20 / truffle 5.9.1
+    # Use an explicit 'paris' EVM target meanwhile, which was the previous default
+    inreplace "truffle-config.js", %r{//\s*evmVersion:.*$}, "evmVersion: 'paris'"
```

Comment:
- this is actually supported in 2.2.1, https://github.com/crytic/echidna/pull/1090